### PR TITLE
Add automatic container restart to Traefik example

### PR DESCRIPTION
**Pull request by t3easy**
Also switch to volume as it is more flexible than recursive path and add examples for external name

See: https://github.com/kereis/traefik-certs-dumper#automatic-container-restart

### CODE REVIEW

File: docs/post_installation/firststeps-rp.de.md [hunk 1]
```diff
@@ -212,14 +212,15 @@ services:
...
     certdumper:
         image: humenius/traefik-certs-dumper
-        container_name: traefik_certdumper
+        command: --restart-containers ${COMPOSE_PROJECT_NAME}-postfix-mailcow-1,${COMPOSE_PROJECT_NAME}-nginx-mailcow-1,${COMPOSE_PROJECT_NAME}-dovecot-mailcow-1
         network_mode: none
         volumes:
-          # mounten Sie den Ordner, der Traefiks `acme.json' Datei enthält
-          # in diesem Fall wird Traefik von seinem eigenen docker compose in ../traefik gestartet
-          - ../traefik/data:/traefik:ro
+          # Bilden Sie das Volume, das Traefiks `acme.json' Datei enthält, ein
```
Comment (DerLinkman) [inline docs/post_installation/firststeps-rp.de.md:218]: Its called "Binden" not "Bilden".

File: docs/post_installation/firststeps-rp.de.md [hunk 2]
```diff
@@ -212,14 +212,15 @@ services:
...
+          - acme:/traefik:ro
           # SSL-Ordner von mailcow einhängen
           - ./data/assets/ssl/:/output:rw
+          # Binde den Docker socket ein um die Container neu zu starten
```
Comment (DerLinkman) [inline docs/post_installation/firststeps-rp.de.md:222]: "Binden Sie den Docker Socket ein, damit Traefik die Container neu starten kann"

**Review by DerLinkman — CHANGES_REQUESTED** [review]
See comments

File: docs/post_installation/firststeps-rp.de.md (hunk 2 above)
Comment (t3easy) [reply docs/post_installation/firststeps-rp.de.md:222]: Thanks, will fix the typo later!

**Review by DerLinkman — APPROVED** [review]
Looks good now! Thank you

**Pull request merged**